src/pages/auth/ForgotPassword.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Trans, t } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+
+import Button from 'components/button/Button';
+import TextField from 'components/textfield/TextField';
+import { Panel, PanelContent } from 'components/panel/Panel';
+import { SignUp, TitleMessage } from './Utilities';
+import { ReactComponent as HandImage } from './assets/images/forgot-password.svg';
+
+const SendResetLink = (props) => (
+  <PanelContent>
+    <div className="flex mb-6">
+      <h2 className="flex-1 text-2xl font-semibold text-primary">
+        <Trans id="auth.signup.recovery" defaults="Forgot Password" />
+      </h2>
+    </div>
+    <div className="block text-secondary-light mb-1">
+      <Trans
+        id="auth.signin.recoverycard"
+        defaults="Your recovery email has been sent! Head on over to your email to reset your password and get back to Coding Coach!"
+      />
+    </div>
+    <div className="flex justify-center content-end">
+      <HandImage />
+    </div>
+  </PanelContent>
+);
+
+const GetAccountInfo = ({ noinfo, clickHandler, saveEmail }) => (
+  <PanelContent>
+    <div className="flex mb-6">
+      <h2 className="flex-1 text-2xl font-semibold text-primary">
+        <Trans id="auth.signup.recovery" defaults="Forgot Password" />
+      </h2>
+    </div>
+    <I18n>
+      {({ i18n }) => (
+        <div>
+          <TextField
+            label={i18n._(t('auth.signup.email')`Email Address`)}
+            name="email"
+            onChange={saveEmail}
+          />
+        </div>
+      )}
+    </I18n>
+    {noinfo ? (
+      <div className="text-sm text-danger mb-2">
+        <Trans
+          id={noinfo ? 'auth.signin.recoveryerror' : ''}
+          defaults="We need this information to find your account."
+        />
+      </div>
+    ) : (
+      <div className="h-6" />
+    )}
+    <Button onClick={clickHandler} size="small" typography="none" fullWidth>
+      <Trans id="auth.signin.help" defaults="Send Help!" />
+    </Button>
+  </PanelContent>
+);
+
+class ForgotPassword extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { displaySuccess: false, email: '', noinfo: false };
```

Comment:
Just for code consistency across the project, remove the `constructor` and define the state like this:

```
class ForgotPassword extends React.Component {
  state = {
    displaySuccess: false,
    email: '',
    noinfo: false,
  };

  //.....
```

Reply:
Sure. Updated!